shared/src/main/scala/tastyquery/Contexts.scala

```diff
@@ -188,14 +188,10 @@ object Contexts {
 
     def moduleRoot: RegularSymbol = Contexts.moduleRoot(classRoot)(using this)
 
-    def createSymbol[T <: Symbol](name: Name, factory: SymbolFactory[T], addToDecls: Boolean = false): T =
-      owner.getDeclInternal(name) match
```

Comment:
is there a reason to get rid of this guard? I guess its for overloads

Reply:
Yes, it's for overloads.